src/Utils/util.cpp

```diff
@@ -23,7 +24,7 @@ void Util::Log(char const * const format, ...)
     } else {
         logFile = fopen(Util::logFileName, "a"); // append to log
     }
-    setbuf( logFile, nullptr ); // Turn off buffered I/O, decreases performance but if crash occurs, no unflushed buffer.
+    setbuf( logFile, nullptr ); // Turns off buffered I/O; decreases performance but if crash occurs, but no unflushed buffer.
```

Comment:
This sounds like it was written by a stroke victim now

Reply:
Yikes, was likely way too high while writing that. Fixed.